static/app/views/relocation/getStarted.tsx

```diff
@@ -43,25 +42,32 @@ function GetStarted(props: StepProps) {
               'In order to best facilitate the process some basic information will be required to ensure sucess with the relocation process of you self-hosted instance'
             )}
           </p>
-          <RequiredLabel>{t('Organization Slugs')}</RequiredLabel>
+          <RequiredLabel>{t('Organization slugs being relocated')}</RequiredLabel>
           <Input
             type="text"
-            name="org-slugs"
+            name="orgs"
             aria-label="org-slugs"
             onChange={evt => setOrgSlugs(evt.target.value)}
             required
-            minLength={1}
-            placeholder=""
+            minLength={3}
+            placeholder="org-slugs-here"
           />
-          <Label>{t('Choose a Datacenter Region')}</Label>
+          <Label>{t('Choose a datacenter region')}</Label>
           <RegionSelect
             value={region}
             name="region"
             aria-label="region"
+            placeholder="Select Region"
             options={regions.map(r => ({label: r.name, value: r.name}))}
             onChange={opt => setRegion(opt.value)}
           />
-          <ContinueButton disabled={!orgSlugs} size="md" priority="primary" type="submit">
+          {region && <p>{t('This is an important decision and cannot be changed.')}</p>}
```

Comment:
Let's leave a `TODO(getsentry/team-ospo#214)` to make this a popup. We want users to fully acknowledge that this is super important and difficult to change.

Reply:
done